src/pick.c

```diff
@@ -95,7 +95,7 @@ static struct {
 static FILE		*tty_in, *tty_out;
 static char		*query;
 static size_t		 query_size;
-static int		 descriptions;
+static int		 descriptions, tty_lines;
```

Comment:
I can't really tell the difference between `tty_lines` and `lines` just from this name.

Would something like `choice_lines` make it clearer that it is `lines` minus 1 for the query line?